recipes/pipreqs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pipreqs" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "06b1589e54bc82233a444f313b344395ddb08079d66511190b3a463e04676221" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pipreqs=pipreqs.pipreqs:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docopt
+    - yarg
```

Comment:
Please drop run time dependencies from build if possible (excepting `python` and potentially `setuptools`).